src/core/CMakeLists.txt

```diff
@@ -1134,6 +1134,12 @@ SET(QGIS_CORE_HDRS
   mesh/qgstriangularmesh.h
   mesh/qgsmeshtracerenderer.h
 
+  pal/pal.h
```

Comment:
Do we really need these ones? pal should be completely internal and not exposed anywhere

Reply:
It is included by `qgslabelfeature.h` [1].

https://github.com/qgis/QGIS/blob/master/src/core/labeling/qgslabelfeature.h#L26

Reply:
Ok, I should be able to fix that

Reply:
@manisandro see https://github.com/qgis/QGIS/pull/33803

Reply:
@nyalldawson Thanks!